src/SixLabors.Fonts/GlyphRendererParameters.cs

```diff
@@ -85,36 +83,27 @@ internal GlyphRendererParameters(GlyphInstance glyph, float pointSize, Vector2 d
             => !left.Equals(right);
 
         /// <inheritdoc/>
-        public bool Equals(GlyphRendererParameters p)
+        public bool Equals(GlyphRendererParameters other)
         {
-            return p.PointSize == this.PointSize &&
-                p.FontStyle == this.FontStyle &&
-                p.DpiX == this.DpiX &&
-                p.DpiY == this.DpiY &&
-                p.GlyphIndex == this.GlyphIndex &&
-                ((p.Font == null && this.Font == null) ||
-                (p.Font?.Equals(this.Font, StringComparison.OrdinalIgnoreCase) == true));
+            return other.PointSize == this.PointSize
+                && other.FontStyle == this.FontStyle
+                && other.DpiX == this.DpiX
+                && other.DpiY == this.DpiY
+                && other.GlyphIndex == this.GlyphIndex
+                && ((other.Font == null && this.Font == null)
+                || (other.Font?.Equals(this.Font, StringComparison.OrdinalIgnoreCase) == true));
         }
 
         /// <inheritdoc/>
-        public override bool Equals(object obj)
-        {
-            if (obj is GlyphRendererParameters p)
-            {
-                return this.Equals(p);
-            }
-
-            return false;
-        }
+        public override bool Equals(object obj) => obj is GlyphRendererParameters p && this.Equals(p);
 
         /// <inheritdoc/>
         public override int GetHashCode()
         {
             var hash = this.Font?.GetHashCode() ?? 0;
```

Comment:
we should probably lowercase the font name when hash coding to match the `OrdinalIgnoreCase` matching in the equals.